bom_stock/__init__.py

```diff
@@ -18,6 +18,6 @@
 #
```

Comment:
@SodexisTeam Thanks for your contribution to the OCA. 
Please follow the guidelines from:
https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md
You need to create a README file following: https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst
Besides, for all your py files, please use short headers following:
https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py#L3